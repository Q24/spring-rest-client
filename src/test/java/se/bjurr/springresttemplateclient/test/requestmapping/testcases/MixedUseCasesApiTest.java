package se.bjurr.springresttemplateclient.test.requestmapping.testcases;

import org.junit.Test;
import se.bjurr.springresttemplateclient.test.requestmapping.spec.api.MixedUseCasesApi;
import se.bjurr.springresttemplateclient.test.utils.BaseApiTest;

public class MixedUseCasesApiTest extends BaseApiTest<MixedUseCasesApi> {

  @Override
  public Class<MixedUseCasesApi> getSutClass() {
    return MixedUseCasesApi.class;
  }

  @Test(expected = RuntimeException.class)
  public void doGetNoNameInPathVariable() {
    this.getSut().doGetNoNameInPathVariable(1L);
    this.verify();
  }

  @Test
  public void doGetPath() {
    this.getSut().doGetPath(1L);
    this.verify();
  }

  @Test
  public void doGetNoPath() {
    this.getSut().doGetNoPath();
    this.verify();
  }

  @Test
  public void doGetSlashPath() {
    this.getSut().doGetSlashPath();
    this.verify();
  }
}
