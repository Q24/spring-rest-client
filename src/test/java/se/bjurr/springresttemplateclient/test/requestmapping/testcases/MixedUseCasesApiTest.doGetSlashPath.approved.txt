[ {
  "url" : "http://localhost:X/base/",
  "method" : "GET",
  "headers" : {
    "Accept" : "application/json, application/*+json",
    "Connection" : "keep-alive"
  },
  "cookies" : { },
  "body" : ""
} ]